include/kernel.h

```diff
@@ -1842,7 +1842,7 @@ __syscall void k_queue_cancel_wait(struct k_queue *queue);
  * @brief Append an element to the end of a queue.
  *
  * This routine appends a data item to @a queue. A queue data item must be
- * aligned on a 4-byte boundary, and the first 32 bits of the item are
+ * aligned on a word (4 or 8 byte) boundary, and the first word of the item is
```

Comment:
I'd personally find "aligned on a word boundary" (without reference to a specific number of bytes) to be better, here and elsewhere. (Because in 20 years, kids will smile seeing such references from the dark ages of IT ;-) ).